### ISSUE TITLE
No results returned from query

### ISSUE THREAD

**Issue by kody-actzero** [OElesin/querypal#5]
I have forked this project (left everything intact, just fixed some bugs in the cloudformation, etc). I have the app successfully deployed, and can see my athena tables. I can run a query, and the network call shows that it is successful (as well as the query result in S3), but the results are not showing in the UI. It returns the message: "No Data Available". Any idea what might be happening here?

**Comment by OElesin** [comment]
@kody-actzero , thanks for using this. Firstly, do you mind contributing the fixes so everyone can benefit from it?

This is quite strange as well. Is your glue data catalog managed by AWS Lake Formation? If so, you may need to add lake formation permissions. 
However, can you log into your AWS console, goto the Athena console and check query history? This way you can confirm that the queries executed were successful.

**Comment by kody-actzero** [comment]
Hi, 

Sure, once I deploy this to a more suitable environment, I will note the specific changes needed. As a workaround, I granted more permissions than necessary, so will scope them back down when/if I deploy this to a live environment.

I checked in Athena, and the query is successful. We are not using LakeFormation, so we don't need those permissions.

It seems the issue is just displaying the results of the query in the UI. Hoping it is a simple fix.
